Source/Utilities/StringContains.lua

```diff
@@ -0,0 +1,15 @@
+function Auctionator.Utilities.StringContains(s, sub, ...)
+  if s == nil or sub == nil or sub == "" then
+    return false;
+  end
+
+  local start, stop = string.find(string.lower(s), string.lower(sub), 1, true);
+
+  local found = (start ~= nil);
```

Comment:
Remove temp variable (and is the if statement needed?):
```lua
return start ~= nil or select("#", ...) == 0 or Auctionator.Utilities.StringContains(s, ...)
```

Reply:
See previous comment

Reply:
Yeah same here. I would probably just try to capture that first clause as a well named function (which seems like it would encapsulate lines 6-8).